Project/Source Files/Screens/Recipies/RecipiesTableViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  RecipiesViewController.swift
+//  NetguruCollegeApp
+//
+
+import UIKit
+
+class RecipiesTableViewController: UIViewController {
+
+    @IBOutlet var tableView: UITableView!
+    var dataBase = [
+        Recipe(image: UIImage(named: "TestingImage")!,
+               title: "Let's cook something",
+               ingredients: "Garlic, beef, salt, Garlic, beef, salt, Garlic, beef, salt"),
+        Recipe(image: UIImage(named: "TestingImage2")!,
+               title: "Another cool recipe",
+               ingredients: "Mango, Melon, Water")
+    ]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+    }
+    private func setupView() {
+        tableView.register(UINib(nibName: "RecipiesTableViewCell", bundle: nil),
+                           forCellReuseIdentifier: "RecipiesTableViewCell")
+        tableView.delegate = self
+        tableView.dataSource = self
+        NSLayoutConstraint.activate([
+            tableView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
+            tableView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            tableView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor),
+            tableView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor)])
+    }
+}
+
+extension RecipiesTableViewController: UITableViewDelegate, UITableViewDataSource {
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return dataBase.count
+    }
+    func tableView(_ tableView: UITableView,
+                   cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+
+        let cell = tableView.dequeueReusableCell(withIdentifier: "RecipiesTableViewCell",
+                                                 for: indexPath) as! RecipiesTableViewCell
+        cell.recipeImage.image = dataBase[indexPath.row].image
+        cell.recipeTitle.text = dataBase[indexPath.row].title
+        cell.recipeIngredients.text = dataBase[indexPath.row].ingredients
+        return cell
+    }
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 120
+    }
+
+}
+
+
+func createAttributedIngredientsLabel(ingredients: String)->NSMutableAttributedString {
+    let boldText = "Ingredients: "
+    let attributes = [NSAttributedString.Key.font : UIFont.boldSystemFont(ofSize: 15)]
+    var attributedString = NSMutableAttributedString(string:boldText, attributes:attributes)
+    
+    let normalText = ingredients
+    let normalString = NSMutableAttributedString(string:normalText)
+    attributedString.append(normalString)
+    return attributedString
+}
+
+public class Recipe {
```

Comment:
// TODO Would be nice.
Also, did we ask Zosia about how the Recipe model will look like?
Maybe it would be different, then it will take us time to refactor everything.